AutomatedTesting/Gem/PythonTests/editor/EditorScripts/EntityOutliner_EntityOrdering.py

```diff
@@ -78,7 +77,7 @@ def create_entity():
             entity_outliner, "Create entity", index=get_root_prefab_container_index()
         )
         # Wait a tick after entity creation to let events process
-        general.idle_wait(0.0)
+        wait_for_propagation()
```

Comment:
this is just a wrapper for azlmbr.legacy.general.idle_wait_frames(1). this change just moves the import to prefab_utils

Reply:
Yes.

Reply:
Reverted this change due to an Editor crash on Jenkins.

Reply:
Now restored it again after figuring out the actual root cause of the crash.